style.css

```diff
@@ -354,3 +354,99 @@ li {
   font-size: 15px;
   line-height: 20px;
 }
+
+.contact-container {
+  background-image:
+    url("./images/Contact-me-Background/Ellipse\ 3.svg"),
+    url("./images/Contact-me-Background/Ellipse\ 4.svg"),
+    url("./images/Contact-me-Background/Ellipse\ 5.svg"),
+    url("./images/Contact-me-Background/Ellipse\ 8.svg"),
+    url("./images/Contact-me-Background/Ellipse\ 11.svg"),
+    url("./images/Contact-me-Background/Ellipse\ 12.svg"),
+    url("./images/Contact-me-Background/Ellipse\ 13.svg");
+  background-repeat: no-repeat;
+  background-position:
+    95% 9.2%,
+    63.2% 91.8%,
+    35.5% 9.6%,
+    57% 15%,
+    4% 12.6%,
+    91% 76.4%,
+    88.6% 75.4%,
+    79.5% 44%,
+    14.1% 90%,
+    51.5% 91.2%,
+    40% 9.1%;
```

Comment:
- [ ] Please remove the gap between the background images and the edge of the viewport. It is not like that in the design. 
<p align="center">&nbsp;
 
| Figma Design |   Your Project |
| ---------- | ----------- |
 | ![Template 1-Mobile-Main](https://user-images.githubusercontent.com/12524453/177424694-10728e3f-49fc-483d-945d-e74b10f10a11.png) | ![Screen Shot 2022-07-06 at 01 02 50](https://user-images.githubusercontent.com/12524453/177424749-0735aafd-0299-473a-a067-79c7cbabc199.png) |
</p>

Reply:
Thanks for the awesome review so far [wuletawwonte](https://github.com/wuletawwonte), but from the code, the is no gap between the background image and contact area. Nevertheless some adjustments would be made to recreate a more similar thing. Thanks again